launcher/ui/pages/instance/OtherLogsPage.cpp

```diff
@@ -46,12 +46,38 @@
 #include <QShortcut>
 #include "RecursiveFileSystemWatcher.h"
 
-OtherLogsPage::OtherLogsPage(QString path, IPathMatcher::Ptr fileFilter, QWidget* parent)
-    : QWidget(parent), ui(new Ui::OtherLogsPage), m_path(path), m_fileFilter(fileFilter), m_watcher(new RecursiveFileSystemWatcher(this))
+OtherLogsPage::OtherLogsPage(InstancePtr instance, IPathMatcher::Ptr fileFilter, QWidget* parent)
+    : QWidget(parent)
+    , ui(new Ui::OtherLogsPage)
+    , m_instance(instance)
+    , m_path(instance->getLogFileRoot())
+    , m_fileFilter(fileFilter)
+    , m_watcher(new RecursiveFileSystemWatcher(this))
+    , m_model(new LogModel())
```

Comment:
I think you need to set the parent of the model otherwise it won't be destructed